sdk/communication/Azure.Communication.PhoneNumbers/tests/Infrastructure/PhoneNumbersClientLiveTestBase.cs

```diff
@@ -18,6 +19,15 @@ public PhoneNumbersClientLiveTestBase(bool isAsync) : base(isAsync)
         public bool IncludePhoneNumberLiveTests
             => TestEnvironment.Mode == RecordedTestMode.Playback || Environment.GetEnvironmentVariable("INCLUDE_PHONENUMBER_LIVE_TESTS") == "True";
 
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            if (TestEnvironment.ShouldIgnorePhoneNumbersTests)
+            {
+                Assert.Ignore("Phone number tests are skipped.");
```

Comment:
nit: Maybe we can use a more descriptive message so if someone saw it and do not know why, it would help them figure out what is up

Reply:
sure, will change the message